wandb/sdk/lib/service/service_process.py

```diff
@@ -87,12 +88,11 @@ def _launch_server(settings: Settings) -> ServiceProcess:
         if dcgm_profiling_enabled():
             service_args.append("--enable-dcgm-profiling")
 
-        service_args += [
-            "--port-filename",
-            str(port_file),
-            "--pid",
-            pid,
-        ]
+        service_args.extend(["--port-filename", str(port_file)])
+        service_args.extend(["--pid", pid])
+
+        if not ipc_support.SUPPORTS_UNIX:
+            service_args.append("--listen-on-localhost")
```

Comment:
don't we want localhost for all for now?

Reply:
discussed offline! keeping as is: we expect that if SUPPORTS_UNIX is true _and_ wandb-core managed to listen on a Unix socket, then the Python client will be able to connect